x/txfees/types/expected_keepers.go

```diff
@@ -42,19 +38,6 @@ type FeegrantKeeper interface {
 // BankKeeper defines the contract needed for supply related APIs (noalias)
 type BankKeeper interface {
 	SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
-	GetAllBalances(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
 	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
 	SendCoinsFromModuleToModule(ctx sdk.Context, senderModule, recipientModule string, amt sdk.Coins) error
 }
-
-// TxFeesKeeper defines the expected transaction fee keeper
-type TxFeesKeeper interface {
-	ConvertToBaseToken(ctx sdk.Context, inputFee sdk.Coin) (sdk.Coin, error)
-	GetBaseDenom(ctx sdk.Context) (denom string, err error)
-	GetFeeToken(ctx sdk.Context, denom string) (FeeToken, error)
-}
```

Comment:
Lets keep this one, so other modules can import it, rather than having to redefine it locally